### DISCUSSION TITLE
Advanced error message for "Email cannot be sent to {recipient-address} from your alias {sender-alias}"

### DISCUSSION THREAD

**Discussion by techybit** [simple-login/app#729] [Feature Requests]
Hello,
currently I am using the hosted Version of SimpleLogin (not selfhosted)
A few days ago I got my first "Email cannot be sent to {recipient-address} from your alias {sender-alias}" error message.

Currently the message looks like this:
> This might mean {recipient-address}
> • is not a valid email address, or
> • doesn't exist, or
> • its mail server refuses your email
> {Button:} View the original email

I think it would be extremely helpful if the error email or the page with the rejected messages would contain an extended error message.
This would be the error message sent by the destination mail server (e.g. Diagnostic code: smtp;550 5.1.1 <>: Recipient address rejected: User unknown in virtual mailbox table).
It would be extremely helpful to find out, for example, if the e-mail was rejected because of a buggy spam filter or because the address contains a typo.
Of course it would also be possible to create a setting which activates such extended error messages.

**Comment by xairoo** [comment]
Yeah of course this is important!

Right now I have to check the full mail content @ `sl/upload/refused-emails/full-xxx.eml`.

But this mail should just be attached to the notification mail. Maybe there is an option, but I haven't found it yet.
